CONTRIBUTING.md

```diff
@@ -1,25 +1,47 @@
+---
+editor: 
+  markdown: 
+    wrap: 72
+---
```

Comment:
Not sure this has worked? 
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/1cfbd87f-80cc-4890-9910-101a84a08ff3)

I previewed it by switching to the branch then selecting the file, leading me to https://github.com/dfe-analytical-services/analysts-guide/blob/rap-page-additions/CONTRIBUTING.md